samza-kafka/src/test/java/org/apache/samza/system/kafka/TestKafkaSystemAdminJava.java

```diff
@@ -54,7 +55,7 @@ public void testCreateCoordinatorStreamDelegatesToCreateStream() {
   public void testCreateChangelogStreamDelegatesToCreateStream() {
```

Comment:
Suggest updating the name for this test, since createChangelogStream no longer exists so the delegation is not needed. Instead, this test seems to primarily verify that the changelog properties are used when the StreamSpec is a changelog StreamSpec.